src/logger.rs

```diff
@@ -61,6 +63,15 @@ fn should_log(record: &Record) -> bool {
 
 fn open_logfile() -> File {
     let mut path = std::env::current_dir().unwrap();
+    if cfg!(target_os = "macos") {
+        match get_instances_dir() {
```

Comment:
This will work, but we should probably use `AppDirs::new(Some("ESLauncher2"), AppUI::Graphical)?.data_dir` instead. Logs are not instance-specific and have no place in `$data_dir/instances`

Reply:
I found the crate app_dirs which could be used for that. However, can't we unify the approach again for all platforms? Writing to the current directory works only so long as the users don't copy the program to one of the intended places (for Mac or Windows).